src/components/profiles-modals/ImportProfileModal.vue

```diff
@@ -212,17 +212,17 @@ export default class ImportProfileModal extends mixins(ProfilesMixin) {
     async importProfile(targetProfileName: string, mods: ExportMod[], zipPath: string) {
         this.activeStep = 'PROFILE_IS_BEING_IMPORTED';
         this.importPhaseDescription = 'Downloading mods: 0%';
-        const progressCallback = (progress: number|string) => typeof progress === "number"
```

Comment:
IDK if these changes are really needed after the last round changes in other files?